client/src/components/__tests__/Sidebar.test.tsx

```diff
@@ -35,11 +36,15 @@ describe("Sidebar Environment Variables", () => {
   };
 
   const renderSidebar = (props = {}) => {
-    return render(<Sidebar {...defaultProps} {...props} />);
+    return render(
+      <TooltipProvider>
```

Comment:
don't like doing this, let me know if there is any other way of wrapping the test component with a provider.